### PR TITLE
[resotoshell][fix] Fix sort suggestion and improve option information

### PR DESCRIPTION
**Pull request by aquamatthias**
# Description

# To-Dos

- [x] Add test coverage for new or updated functionality
- [x] Lint and test with `tox`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

**Review by lloesche — DISMISSED** [review]
lgtm, thank you 🚢

File: resotocore/resotocore/cli/command.py [hunk 1]
```diff
@@ -3821,19 +3821,16 @@ async def welcome() -> str:
                 else 0
             )
             center_vertical = (
-                int((ctx.console_renderer.height - 21) / 2)
+                max(0, int(ctx.console_renderer.height - 27))
```
Comment (meln1k) [inline resotocore/resotocore/cli/command.py:3824]: what do these magic constants do?

File: resotoshell/resotoshell/promptsession.py [hunk 2]
```diff
@@ -118,13 +123,22 @@ def get_completions(
         return self.fuzzy_completer.get_completions(document, complete_event)
 
     def get_completions_filtered(
-        self, document: Document, complete_event: CompleteEvent, **filter_values: bool
+        self,
+        document: Document,
+        complete_event: CompleteEvent,
+        fn: Callable[[Completion], bool],
```
Comment (meln1k) [inline resotoshell/resotoshell/promptsession.py:129]: What about calling fn predicate?

**Review by meln1k — APPROVED** [review]
LGTM

**Pull request merged**